idrisadetunmbi/github-android-developers
# #155849507 API endpoints set up completion

#### What does this PR do?
- Complete the API endpoint and model/presenter setups for retrieving the list of users.

#### Description of Task to be completed?
- Create Java classes for the resources
- Create the retrofit instance
- Define the API endpoints
- Accessing the API

#### How should this be manually tested?
Clone the repo in android studio, run the app and check the fetched users in the log.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#155849507
#155892103

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
